docs/src/administration_configuration.md

```diff
@@ -5,7 +5,12 @@ The configuration is based on the file
 This file also contains documentation for all the available options. To override the defaults, you
 can copy the options you want to change into your local `config.hjson` file.
 
-To use a different `config.hjson` location than the current directory, set the environment variable `LEMMY_CONFIG_LOCATION`. Make sure you copy the `defaults.hjson` if you do this, otherwise you will be missing settings.
+The `defaults.hjson` and `config.hjson` files are located at `config/defaults.hjson` and
+`config/config.hjson`, respectively. To change these default locations, you can set these
+two environment variables:
+
+    LEMMY_CONFIG_LOCATION           # config.hjson
+    LEMMY_CONFIG_DEFAULTS_LOCATION  # defaults.hjson
```

Comment:
There's some weird line breaking and indentation formatting here. This is a markdown file so that's not necessary.

Reply:
I added the indentation because it's a markdown file, I figured it would be good to have those environment variables visually called out in the documentation when someone's scanning through it. I can change that to something else though if you'd like.

As far as the line breaking, my default line length when typing is apparently longer than Github's max line length for word wrapping, hence the weird wrapping. At any rate, just removed the manual line breaks, since you're right that it's going to be rendered via markdown anyways.